### DISCUSSION TITLE
Templates for NSF and/or NIH proposals?

### DISCUSSION THREAD

**Discussion by jonmatthis** [jupyter-book/mystmd#1453] [General]
Hello! Just found this project and I'm excited to explore!

I was wondering if anyone had put any work into making templates for standard NSF/NIH/etc proposals? 

I'm thinking like a template that follows the different layouts in repos like these: 

- https://github.com/nsf-open/nsf-proposal-latex-samples 
- https://github.com/ntjess/typst-nsf-templates

**Comment by jonmatthis** [comment]
Note - I found the discord server after making this post and moved the discussion to a post in the `Support` forum: https://discord.com/channels/1083088970059096114/1272967621813338153

**Reply by rowanc1** [reply]
Thanks @jonmatthis -- excited to help you out with getting the NIH/NSF templates working!

For other folks seeing this, there are also some template tutorials here:
https://mystmd.org/jtex/create-a-latex-template
